packages/headless/src/controllers/commerce/context/cart/headless-cart.ts

```diff
@@ -37,6 +37,12 @@ export interface CartItem {
    */
   productId: string;
 
+  /**
+   * The unique identifier of the product's variant.
+   * If the item is not a variant, set the same value as the productId.
```

Comment:
Could it be clearer if we rephrased to:

```
The stock keeping unit for the item being added to cart.
Depending on how your catalog is structured, this may be the same value as the productId.
```

Reasoning:
- sku implies a unique identifier, so there isn't a need to specify this explicitly
- By asking for a "stock keeping unit" we are guiding the implementer to pass a value that semantically means "sku" rather than a different unique identifier.
- Depending on how the catalog is structured, this might be the same value as `productId`. However, it is possible for a customer to store a unique identifier in `permanentid` (productId) field, and to also send the sku in `ec_sku`, in a products-only catalog. Not sure if this is happening in practice, but being vague ("it depends on the catalog"), we are guiding the reader to send a sku, while still reassuring them that it's ok if it's the same value as the `productId` field.